File: modul-4/PRAK304-2210817210012-MuhammadRakaAzwar.c
#include<stdio.h>
int main()
{
    int bil;
    int i = 1;
    while (i < 2){
            printf ("");
            scanf ("%d", &bil);
            if (bil <0 || bil >=100) {
                    printf("Anda Menginput Melebihi Limit Bilangan.\n");
            }
            else if (bil <=9 && bil >=1) {
                    printf ("Satuan.\n");
            }
            else if (bil <=19 && bil >=11) {
                    printf ("Belasan.\n");
            }
            else if (bil == 0) {
                    printf("Nol.\n");
            }
            else
                printf ("Puluhan.\n");
    }
}

File: modul-4/PRAK304-2210817210012-MuhammadRakaAzwar.py
i = 1
while i < 2:
        bil = input()
        bil = int (bil)
        if (bil <=99 and bil >=20 or bil == 10):
                print("Puluhan.")
        elif (bil <=9 and bil >=1):
                print("Satuan.")
        elif (bil <=19 and bil >=11):
                print("Belasan.")
        elif (bil == 0):
                print("Nol.")
        else:
                print("Anda Menginput Melebihi Limit Bilangan.")